packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -49,9 +49,30 @@ exports.sourceNodes = async (
     disallowedLinkTypes,
     skipFileDownloads,
     fastBuilds,
+    translation,
   } = pluginOptions
   const { createNode, setPluginStatus, touchNode } = actions
 
+  // Default apiBase to `jsonapi`
```

Comment:
Had to move default values further up in the process to allow access to them when fetching language configuration.